xpsi/PostProcessing/_corner.py

```diff
@@ -497,7 +497,7 @@ def _plot_triangle(self,
         if scale_attrs is None:
             scale_attrs = {}
 
-        for key, value in scale_attrs.items():
+        for key, value in list(scale_attrs.items()):
```

Comment:
Same here, IIRC part of why these methods were turned into generators is so that you don't *have* to allocate space in memory for the entire list object. So turning them into lists might defeat the purpose? I might be wrong, though.